02-technical-guides/hierarchies-advanced/main.ipynb

```diff
@@ -0,0 +1,1998 @@
+{
```

Comment:
<p>@tibdex I noticed that when I set my <code>OrderId</code> hierarchy as a virtual hierarchy, I notice and <strong>increase</strong> in memory footprint rather than a <strong>decrease</strong>. Perhaps this is just specific to my environment? 🤷‍♂️ When you get a chance, could you try to run through this section of the notebook and reproduce the issue? Not sure why this would be the case.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/atoti/atoti/pull/890/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='62'/>